templates/components/news-list/news-list.ui_patterns.yml

```diff
@@ -38,7 +34,7 @@ news-card:
       preview: "I am a link"
   use: "@news/news-list/news-item-list.twig"
   libraries:
-    - card:
```

Comment:
This library gets prefixed with all sorts of namespacey stuff so it doesn't really matter but we should be good about naming things.